### PR TITLE
Update pim docs 2.0 for the projects console

### PR DESCRIPTION
**Pull request by Nuscly**
**Description**

The main goal is to update the old reference in technical documentation to use the Portal section of the Help Center for 2.0.

On a serenity mode, it adds the documentation on how to set up your instance with the Projects Console in the Portal.

Note: on this PR, the documentation does not reference the Partners Portal but the Portal. Depending if you are a customer or not, you could use the Customer Portal too. This is explained in the Help Center.

PR linked to https://akeneo.atlassian.net/browse/PAPO-1514 on the Portal.

**Definition Of Done**

| Q                                 | A
| --------------------------------- | ---
| Technical Review and 2 GTM        | Todo
| English Review and 1 GTM          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: cloud_edition/serenity_mode/docs/overview.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
++--------------------------------------+--------------------------------------+
+| - The **production** environment:    | my-project.cloud.akeneo.com          |
++--------------------------------------+--------------------------------------+
+| - The **sandbox** environment:       | my-project-staging.cloud.akeneo.com  |
++--------------------------------------+--------------------------------------+
+
+.. warning::
+
+    **Warning:** Only the production environment is backuped. All data produced on the sandbox environment are not backed up.
```
Comment (rybus) [inline cloud_edition/serenity_mode/docs/overview.rst]: backed up
```suggestion
    **Warning:** Only the production environment is backed up. Any data produced on the staging environment are not
```

File: cloud_edition/serenity_mode/docs/overview.rst [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+.. note::
```
Comment (rybus) [inline cloud_edition/serenity_mode/docs/overview.rst]: seems to obvious to be stated but why not after all ;)

File: install_pim/manual/installation_archive_ee.rst.inc [hunk 3]
```diff
@@ -11,7 +11,7 @@ Extracting the archive
     Replace */path/to/installation* by the path to the directory where you want to install the PIM.
 
 .. note::
-    Replace *pim-enterprise-standard-v2.0.tar.gz* by the location and the name of the archive you have downloaded from `Partner portal <https://partners.akeneo.com/login>`_
+    Replace *pim-enterprise-standard-v2.0.tar.gz* by the location and the name of the archive you have downloaded from `the Portal <https://help.akeneo.com/portal/articles/get-akeneo-pim-enterprise-archive.html?utm_source=akeneo-docs&utm_campaign=installation_archive>`_
```
Comment (rybus) [inline install_pim/manual/installation_archive_ee.rst.inc]: from the Portal. Instructions on how to download this archive:

File: cloud_edition/serenity_mode/docs/overview.rst (hunk 1 above)
Comment (mmetayer) [inline cloud_edition/serenity_mode/docs/overview.rst]:
```suggestion
    **Warning:** Only the production environment is backed up. All data produced on the sandbox environment are not backed up.
```

File: cloud_edition/serenity_mode/docs/overview.rst [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+--------
+
+To start, you need to set up your serenity instances from the `Portal <https://help.akeneo.com/portal/articles/set-up-akeneo-serenity.html?utm_source=akeneo-docs&utm_campaign=serenity_overview>`_
+
+Akeneo Cloud Serenity provides 2 environments:
+
++--------------------------------------+--------------------------------------+
+| - The **production** environment:    | my-project.cloud.akeneo.com          |
++--------------------------------------+--------------------------------------+
+| - The **sandbox** environment:       | my-project-staging.cloud.akeneo.com  |
```
[the remark this answers is not in the document]
Comment (Nuscly) [reply cloud_edition/serenity_mode/docs/overview.rst]: Sorry here, but It's intentional: the naming changed, the type of the non-prod instance of the environment is now named "sandbox". I that type of environment Development/Trunk, Integration, Staging, Preprod, etc... See the Project Console documentation.

File: cloud_edition/serenity_mode/docs/overview.rst (hunk 2 above)
Comment (Nuscly) [reply cloud_edition/serenity_mode/docs/overview.rst]: I take as model the flexibility page.

File: cloud_edition/serenity_mode/docs/overview.rst (hunk 4 above)
[the remark this answers is not in the document]
Comment (rybus) [reply cloud_edition/serenity_mode/docs/overview.rst]: Awesome!

**Pull request merged**